src/core/services/layout/layout.js

```diff
@@ -42,25 +44,41 @@
      */
     angular.module('material.core.layout', [ 'ng' ])
 
+      /**
+       * Model of flags used by the Layout directives
+       * Allows changes while running tests or runtime app changes
+       */
+      .factory("$$mdLayout", function() {
+        return {
+          /**
+           * Should we remove the original layout Attribute selectors
+           * after translation injection
+           */
+          removeAttributes : true,
+
+          /**
+           * Special internal flag used to optimize
+           * noop(s) for the directive postLinks below
+           */
+          disablePostLinks : undefined
```

Comment:
because we have not performed a scan/check. After the scan it will be either true or false.